docling/backend/hwp_backend.py

```
@@ -32,69 +30,64 @@ def __init__(self, in_doc: InputDocument, path_or_stream: Union[Path, BytesIO])
                         temp_file.write(path_or_stream.getbuffer())
                     path_or_stream = temp_hwp_path
 
-                # HWP를 HWPX로 변환
                 hwpx_path = self._convert_hwp_to_hwpx(path_or_stream)
-                
-                # HwpxDocumentBackend 인스턴스 생성
                 self.hwpx_backend = hwpx_backend.HwpxDocumentBackend(in_doc, hwpx_path)
                 self.valid = self.hwpx_backend.is_valid()
             except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)